docs/docs/providers/spotify.md

```diff
@@ -0,0 +1,29 @@
+---
+sidebar_label: Spotify
+---
+
+# Spotify API wiki
+
+:::note Working with the Spotify API?
+Please add your learnings, favorite links and gotchas here by [editing this page](https://github.com/nangohq/nango/tree/master/docs/docs/providers/spotify.md).
+
+:::
+
+## Using Spotify with Nango
+
+Provider template name in Nango: `spotify`  
+Follow our [quickstart](../quickstart.md) to add an OAuth integration with Spotify in 5 minutes.
+
+## App registration & publishing
+
+**Rating: `Easy & fast`**
+Registering an app takes only a few minutes, and you can start building immediately: [App registration docs](https://developer.spotify.com/documentation/web-api/concepts/authorization)
+
+
+## Useful links
+
+- [Web API docs (their REST API)](https://developer.spotify.com/documentation/web-api#:~:text=the%20API%20specification-,API%20reference,-The%20Spotify%20Web)
```

Comment:
Let's add the link for `OAuth-related docs` -> https://developer.spotify.com/documentation/web-api/tutorials/code-flow